lib/pechkin/handler.rb

```diff
@@ -31,9 +31,10 @@ def handle(channel_id, msg_id, data)
         prepare_message(channel_id, msg_id, data)
       chats = channel_config.chat_ids
       connector = channel_config.connector
+      email = data["email"]
```

Comment:
This approach will require to have `email` field set. Not all systems can keep this contract and you may want to have it more configurable.